core/store.go

```diff
@@ -1,6 +1,7 @@
 package core
 
 import (
+	"path/filepath"
```

Comment:
Isn't this a heavy library for a basic usecase? A lightweight implementation of the same is possible via ``reflect`` and ``regexp``. Note: ``reflect`` would be needed to deduce the type " INT, FLOAT, DOUBLE, BOOLEAN, STRING", so regex could match the values better. This however could be a non-issue for the time-being.